src/bun.js/bindings/JSValue.zig

```diff
@@ -1573,8 +1573,26 @@ pub const JSValue = enum(i64) {
     ///
     /// For values that are already objects, this is effectively a reinterpret
     /// cast.
-    pub fn toObject(this: JSValue, globalThis: *JSGlobalObject) *JSObject {
-        return cppFn("toObject", .{ this, globalThis });
+    ///
+    /// ## Safety
+    /// Calling `toObject` on `null` or `undefined` values triggers safety-checked
+    /// **Illegal Behavior**.
+    ///
+    /// ## References
+    /// - [ECMA-262 7.1.18 ToObject](https://tc39.es/ecma262/#sec-toobject)
+    pub fn toObject(this: JSValue, globalThis: *JSGlobalObject) callconv(bun.callconv_inline) *JSObject {
```

Comment:
In the future I think we should change this to `JSError!*JSObject` to match JSC, but this is a small change that can be made later